examples/client.c

```diff
@@ -19,6 +19,17 @@ void handler_TheAnswerChanged(UA_UInt32 handle, UA_DataValue *value) {
     return;
 }
 
+UA_StatusCode nodeIter(UA_NodeId childId, UA_Boolean isInverse, UA_NodeId referenceTypeId, void *handle);
```

Comment:
@ichrispa 
The nodes will contain server internals that end users shouldn't see.
For example, the userExecutable flag will not stay a boolean, but be a callback into the user/rights mgmt layer.
Otherwise, we cannot have different rights for different users.

I'd prefer not pushing void pointers with server internals in the client layer.